treeherder/webapp/api/classifiedfailure.py

```diff
@@ -1,6 +1,6 @@
 from collections import defaultdict
 
-import rest_framework_filters as filters
+import django_filters
```

Comment:
We use `rest_framework_filters.OrderingFilter`([source](https://github.com/encode/django-rest-framework/blob/master/rest_framework/filters.py)) in treeherder/webapp/api/performance_data.py below. It is different from `django_filters.rest_framework.OrderingFilter` as can be found out from [here](https://github.com/carltongibson/django-filter/blob/develop/django_filters/filters.py#L583), which is probably why we did not get deprecation warnings for `OrderingFilter` like we did for `DjangoFilterBackend` and `FilterSet` . 

So, I suppose we can modify the `import` in all files other than that .